src/core/connection.c

```diff
@@ -2856,16 +2857,16 @@ QuicConnRecvVerNeg(
         //
         QuicConnCloseLocally(
```

Comment:
You need to close regardless of SupportedVersion being zero or not, because we don't support version negotiation yet.

Reply:
Ah, but then we'll continue to fail the handshake corruption tests in QNS.

Reply:
There's not much else you can do if you receive a VN packet that doesn't include your requested version.